src/main/java/com/hazelcast/remotecontroller/Cluster.java

```diff
@@ -1,13 +1,13 @@
 /**
- * Autogenerated by Thrift Compiler (0.13.0)
+ * Autogenerated by Thrift Compiler (0.15.0)
```

Comment:
Some versions of thrift is now working with .net
We had some problem and it would be better to use a version which we are sure it is working with all clients without any problem.
Most probably 0.16.0 will work without any problems but for now it is better to use 0.15.0 I guess